jungchaeeun/3주차/백준 10995번 문제.py

```diff
@@ -0,0 +1,27 @@
+'''
+문제
+예제를 보고 규칙을 유추한 뒤에 별을 찍어 보세요.
+
+입력
+첫째 줄에 N(1 ≤ N ≤ 100)이 주어진다.
+
+출력
+첫째 줄부터 차례대로 별을 출력한다.
+
+예제 입력 4 
+4
+
+예제 출력 4 
+* * * *
+ * * * *
+* * * *
+ * * * *
+'''
+
+n = int(input())
+
+for i in range(1, n+1, 1):
+    if i % 2 == 0:
+        print(' *' * n)
+    else:
+        print('* ' * n)
```

Comment:
```suggestion
    if i % 2 == 0:
        print(' *' * n)
    else:
        print('* ' * n)
```
짝수 홀수로 나눠서 작성한게 깔끔하네요